src/librustc_typeck/check/op.rs

```diff
@@ -333,8 +333,18 @@ impl<'a, 'gcx, 'tcx> FnCtxt<'a, 'gcx, 'tcx> {
                                 lhs_ty);
 
                             if !lhs_expr.span.eq(&rhs_expr.span) {
-                                self.add_type_neq_err_label(&mut err, lhs_expr.span, lhs_ty);
-                                self.add_type_neq_err_label(&mut err, rhs_expr.span, rhs_ty);
+                                self.add_type_neq_err_label(&mut err,
+                                                            lhs_expr.span,
+                                                            lhs_ty,
+                                                            rhs_ty,
+                                                            op,
+                                                            is_assign);
+                                self.add_type_neq_err_label(&mut err,
+                                                            rhs_expr.span,
+                                                            rhs_ty,
+                                                            lhs_ty,
+                                                            op,
+                                                            is_assign);
```

Comment:
put the arguments in their own line indented four characters, like it is done at https://github.com/rust-lang/rust/pull/59798/files/cbcbd2c3bb52cd6c53c45a6a2476d510c892d2cc..199b0bad3c233ba5bec8895efe6c8888c9ce5bee#diff-18d99e659faf20ef0968cc7bad7f81b2R428